### PR TITLE
Drive-v3: team drive snippets

### PR DESCRIPTION
**Pull request by RajeshGogo**
# Description
Create a drive for team.
Finds all Team Drives without an organizer and add one.
Fixes # (211027701)

## Has it been tested?
- [X] Development testing done
- [X] Unit or integration test implemented

## Checklist
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have performed a peer-reviewed with team member(s)
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: drive/snippets/drive_v3/src/main/java/CreateTeamDrive.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import com.google.api.services.drive.model.TeamDrive;
+import com.google.auth.http.HttpCredentialsAdapter;
+import com.google.auth.oauth2.GoogleCredentials;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.UUID;
+
+/* class to demonstrate use-case of Drive's create team drive. */
+public class CreateTeamDrive {
```
Comment (sqrrrl) [inline drive/snippets/drive_v3/src/main/java/CreateTeamDrive.java:30]: Here and throughout -- TeamDrive => Drive ( the "Team Drive" term is deprecated)

File: drive/snippets/drive_v3/src/main/java/RecoverTeamDrive.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+import com.google.auth.http.HttpCredentialsAdapter;
+import com.google.auth.oauth2.GoogleCredentials;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/* class to demonstrate use-case of Drive's recover all team drives without an organizer. */
+public class RecoverTeamDrive {
```
Comment (sqrrrl) [inline drive/snippets/drive_v3/src/main/java/RecoverTeamDrive.java:33]: Same as other file, change TeamDrive => Drive

File: drive/snippets/drive_v3/src/main/java/CreateTeamDrive.java (hunk 1 above)
Comment (RajeshGogo) [reply drive/snippets/drive_v3/src/main/java/CreateTeamDrive.java:30]: Also [END drive_create_team_drive] => [END drive_create_drive] need to raise CL

File: drive/snippets/drive_v3/src/main/java/RecoverTeamDrive.java (hunk 2 above)
Comment (RajeshGogo) [reply drive/snippets/drive_v3/src/main/java/RecoverTeamDrive.java:33]: Also [END drive_create_team_drive] => [END drive_create_drive] need to raise CL

**Pull request closed without merging**